src/plugins/data/public/data_sources/datasource/types.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { IndexPattern } from '../../index_patterns';
+import { DataSourceType } from '../datasource_services';
+
+export interface IDataSourceMetaData {
+  name: string;
+}
+
+export interface IDataSourceGroup {
```

Comment:
Is this being used anywhere?

Reply:
Will address after this PR merge, Tracking [issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5211).